pom.xml

```diff
@@ -77,6 +77,23 @@
 			<version>${jmeter.version}</version>
 			<scope>test</scope>
 		</dependency>
+		<!-- https://mvnrepository.com/artifact/org.apache.jmeter/ApacheJMeter_components -->
+		<dependency>
+			<groupId>org.apache.jmeter</groupId>
+			<artifactId>ApacheJMeter_components</artifactId>
+			<version>${jmeter.version}</version>
```

Comment:
Should have `provided` scope because we use the `shade` plugin to build a uber jar with the dependencies in it.